src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+}
+
+body {
+  font-weight: 300;
+}
+
+.form {
+  position: relative;
+  color: #3d4e61;
+  width: 100%;
+  margin-top: 20px;
+}
+
+.form--big {
+  height: 70px;
+}
+
+.form--small {
+  height: 42px;
+
+}
```

```suggestion
.form {
  position: relative;
  color: #3d4e61;
  width: 100%;
  margin-top: 20px;
  height: 70px;
}

.form--small {
  height: 42px;
}
```
as example how to set styles without .form--big modifier